README.md

```diff
@@ -59,7 +59,7 @@ After downloading the Opaque codebase, build and test it as follows.
 
 ## Usage
 
-Next, run Apache Spark SQL queries with Opaque as follows, assuming [Spark 3.0.0](https://github.com/apache/spark/releases/tag/v3.0.0) is already installed:
+Next, run Apache Spark SQL queries with Opaque as follows, assuming [Spark 3.0.0](https://www.apache.org/dyn/closer.lua/spark/spark-3.0.1/spark-3.0.1-bin-hadoop2.7.tgz) (`wget http://apache.mirrors.pair.com/spark/spark-3.0.1/spark-3.0.1-bin-hadoop2.7.tgz`) is already installed:
```

Comment:
Can you change `assuming [Spark 3.0.0]` to `assuming [Spark 3.0]`? Otherwise we'll have a bit of inconsistency